json/src/lib.rs

```diff
@@ -1124,6 +1125,29 @@ pub struct SignRawTransactionInput {
     pub amount: Option<Amount>,
 }
 
+#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
+pub struct GetTxOutSetInfoResult {
+    /// The current block height (index)
+    pub height: u64,
+    /// The hash of the block at the tip of the chain
+    #[serde(rename = "bestblock")]
+    pub best_block: bitcoin::BlockHash,
+    /// The number of transactions with unspent outputs
+    pub transactions: u64,
+    /// The number of unspent transaction outputs
+    #[serde(rename = "txouts")]
+    pub tx_outs: u64,
+    /// A meaningless metric for UTXO set size
+    pub bogosize: u64,
+    /// The serialized hash
+    pub hash_serialized_2: sha256::Hash,
```

Comment:
Eventually this might become a hash newtype, what do you think?

Reply:
Yes, this could. But this will need to happen in rust-bitcoin, right?

Reply:
Yeah, that's why I see it as a future improvement, otherwise this would be a blocker until the next release ships. It's probably ok to just open an issue as a reminder.